addons/semicolon/addon.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Semicolon Glitch",
+  "description": "Adds the Semicolon Glitch back to Scratch",
+  "credits": [
+    {
+      "name": "GrahamSH"
+    }
+  ],
+
+  "userstyles": [
+    {
+      "url": "semicolon.css",
+      "matches": ["https://scratch.mit.edu/*"]
+    }
+  ],
+  "userscripts": [
+    {
+      "url": "semicolon.js",
+      "matches": ["https://scratch.mit.edu/*"]
+    }
+  ],
+  "tags": ["community", "forums", "editor"],
```

Comment:
Why forums and editor?

Reply:
Because it does it everywhere, like the original glitch.

Reply:
I don't think adding it to the editor is a good idea. It adds an annoying scrollbar.

Reply:
So did the original glitch, though.

Reply:
It was a glitch, that's why it did :P The new intentional semicolon easter eggs by the ST did not appear in the editor

Reply:
Ok. I'll do it later.